File: 1/solution_1.js

function sumMultiples (){
    const threshold = 1000;
    let sum = 0;
    for(let i = 0;i<threshold;i++){
        if(i % 3 === 0 || i % 5 === 0){
            sum += i 
        }
    }
    return sum
}
console.log(sumMultiples())

File: 1/solution_1.rb
def sumMultiples()
    _THRESHOLD = 1000
    sum = 0
    for i in 0.._THRESHOLD - 1 do
        if i % 3 == 0 || i % 5 == 0 then
          sum += i
        end
    end
    sum
  end
  
puts sumMultiples 